Sources/Testing/Events/Recorder/Event.Symbol.swift

```diff
@@ -0,0 +1,171 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+extension Event {
+  /// An enumeration describing the symbols used as prefixes when recording an
+  /// event.
+  public enum Symbol: Sendable {
```

Comment:
If you do adopt the suggestion elsewhere about a new `@_spi` group, I imagine it ought to be applied here, too